awstransfer-s3-sam/README.md

```diff
@@ -0,0 +1,165 @@
+# Bidirectional selective file transfer between remote SFTP server and Amazon S3 using AWS Transfer Family Connector
+
+This pattern shows how to setup an AWS Transfer Family SFTP connector to list files from the remote server and transfer specific file to Amazon S3 bucket. You can also transfer specific file from Amazon S3 bucket to the remote SFTP Server.  
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/awstransfer-s3-sam. 
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+* [JQ](https://docs.aws.amazon.com/solutions/latest/dynamic-object-and-rule-extensions-for-aws-network-firewall/operation-and-customization.html#install-jq) should be installed.
+
+
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+
+2. Change directory to the pattern directory:
+    ```
+    cd serverless-patterns/awstransfer-s3-sam
+    ```
+
+3. From the command line, run the below command to deploy the pattern:
+    ```
+    bash deploy.sh
```

Comment:
How would a customer using Windows do this?

Reply:
Customer using Windows system can use any command line tool that support using using bash script. I used to use GitBash but there may other alternative options also:
https://stackoverflow.com/questions/26522789/how-to-run-sh-on-windows-command-prompt